hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -220,6 +223,8 @@ genContestMutation
                   (replacePolicyIdWith testPolicyId otherHeadId healthyClosedHeadTxOut)
                   (Just $ toScriptData healthyClosedState)
               ]
+      , SomeMutation (Just "burning is forbidden") MutateTokenBurning
+          <$> changeMintedTokens tx (valueFromList [(AssetId testPolicyId "badTokenBurned", Quantity (-1))])
```

Comment:
Sorry, I don't understand what exactly do you mean here? Let's pair on it?

Reply:
My understanding is that you wouldn't be able to burn or mint in a transaction without providing the corresponding minting policy script. And this test, to the best of my knowledge, does not include this policy in the transaction so I feel like the transaction you test would not work anyway because of this missing.

We can pair on that tomorrow morning 👍 just ping me when you want

Reply:
You are right @pgrange! If it is hard to do that, however, I think it's fine if only the phase-2 check is tested here.

Reply:
If it's hard maybe I would add some comment explaining why we do not include the minting policy eventhough we should in real life.